dns/unbound-plus/src/opnsense/mvc/app/models/OPNsense/Unboundplus/Miscellaneous.xml

```diff
@@ -1,10 +1,18 @@
 <model>
     <mount>//OPNsense/unboundplus/miscellaneous</mount>
     <description>Unbound Miscellaneous configuration</description>
-    <version>0.0.1</version>
+    <version>0.0.2</version>
     <items>
         <privatedomain type="CSVListField">
             <Required>N</Required>
         </privatedomain>
+        <dotservers type="CSVListField">
+            <Required>N</Required>
+            <mask>/^[a-fA-F0-9\.\@]{1,46}$/</mask>
+        </dotservers>
+        <verifydotserver type="BooleanField">
```

Comment:
I'm not sure what the benefit is of making this configurable. It already is configurable via System: Trust: Authorities

Reply:
Oh, I wasn't aware that adding a CA to Authorities also adds it do the /etc/ssl/cert.pem ...

Reply:
Ok, again what learned :) I reverted it and just added cert.pem